bindata/v3.11.0/kube-scheduler/pod.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: openshift-kube-scheduler
+  namespace: openshift-kube-scheduler
+  labels:
+    app: openshift-kube-scheduler
+    scheduler: "true"
+spec:
+  containers:
+  - name: scheduler
+    image: ${IMAGE}
+    imagePullPolicy: Always
+    terminationMessagePolicy: FallbackToLogsOnError
+    command: ["hyperkube", "kube-scheduler"]
+    args:
+    - "--config=/etc/kubernetes/static-pod-resources/configmaps/deployment-kube-scheduler-config/config.yaml"
+    volumeMounts:
+    - mountPath: /etc/kubernetes/static-pod-resources
+      name: resource-dir
+  hostNetwork: true
```

Comment:
this is a change to before: we didn't have host networking for the scheduler afaik.

Reply:
> his is a change to before: we didn't have host networking for the scheduler afaik.

one platform you wouldn't need it, but in a static pod it makes sense because this should be up before the operator that installs the network is available.

Reply:
That makes sense. I have restored it and hopefully that is the reason that kube-scheduler has been failing.